core-bundle/src/Controller/Page/Error503PageController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Controller\Page;
+
+use Contao\CoreBundle\Controller\AbstractController;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\ServiceAnnotation\Page;
+use Contao\FrontendIndex;
+use Contao\PageModel;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Page("error_503", path=false)
+ */
+class Error503PageController extends AbstractController
```

Comment:
The class has essentially the same code as the `Error4xxPageController` class, except the `supportsContentComposition()` method which is missing here IMHO. Do we still need a separate class?

Reply:
Yes, because this one always supports content composition. We could check for the type though.

Reply:
Changed in dd61a4f67876ce024a61eeb319215623e17117e2